main/main.go

```diff
@@ -1,12 +1,19 @@
 package main
 
 import (
+	"context"
 	"github.com/kelseyhightower/envconfig"
 	log "github.com/sirupsen/logrus"
 	"github.com/weihesdlegend/Vacation-planner/planner"
+	"github.com/weihesdlegend/Vacation-planner/utils"
 	"net/url"
+	"os"
+	"os/signal"
+	"sync"
 )
 
+const numWorkers = 5
```

Comment:
Hardcoded ??

Reply:
yes, it's just moved from the planner pkg